src/main/webapp/portal/misc/partials/circle-button.html

```diff
@@ -1,10 +1,8 @@
-<!--<circle-button data-href='' data-target='' data-fa-icon='' data-disabled='false' data-title=''></circle-button>-->
-
 <a ng-href="{{ cbDisabled ? '#' : href}}" target="{{cbDisabled ? '' : target}}">
   <div class="btn btn-primary rounded icon-button text-right" ng-class="{ 'btn-disabled' : cbDisabled}"> 
     <i class="fa {{faIcon}} fa-2x"></i>
   </div>
 </a>
 <p ng-show='title'>
-  {{title}}
-</p>
\ No newline at end of file
+  {{title | truncate:trunclen}}
```

Comment:
If `data-trunclen` is not given then it doesn't truncate and shows the full text. Tested with the list-of-links widget.